pkgs/dash_analytics/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Should the text and/or LICENSE file be updated to match other code under `dart-lang/*`? If not, I can easily change the year to be 2023

Reply:
I think it would simplify things if we did use a consistent license file and copyright blurb for this repo. Can you copy the license file from the repo root into this package (https://github.com/dart-lang/tools/blob/main/LICENSE), and update the file headers to that from https://github.com/dart-lang/tools/blob/main/CONTRIBUTING.md?plain=1#L41?